user_docs/en/changes.md

```diff
@@ -9,6 +9,7 @@
 * Support for math in PDFs has been added.
 This works for formulas with associated MathML, such as some files generated by newer versions of TeX/LaTeX.
 Currently this is only supported in Foxit Reader & Foxit Editor. (#9288, @NSoiffer)
+* Full remote access functionality based on the NVDA Remote add-on has been integrated into core, allowing users to control another computer running NVDA or allow their computer to be controlled remotely for assistance and collaboration. Previously available only as an add-on, this functionality is now built into NVDA with improved security, better integration with NVDA's systems, and enhanced maintainability. (#4390, #17580, @ctoth, @tspivey, @daiverd, NVDA Remote Contributors and funders)
```

Comment:
I would replace "an add-on" with add-ons. We cannot know if just an add-on or avrious ones exist for a featureleast a fo.